pkg/skaffold/build/cluster/kaniko_test.go

```diff
@@ -104,12 +104,20 @@ func TestArgs(t *testing.T) {
 			},
 			shouldErr: true,
 		},
+		{
```

Comment:
we need a test for when `artifactRegistry` has issues parsing the image.

Reply:
done ptal